parsers/discord.py

```diff
@@ -0,0 +1,154 @@
+import functools
+
+from PyQt5 import QtCore
+from PyQt5.QtWebEngineWidgets import QWebEngineView
+from PyQt5.QtWidgets import QScrollArea, QPushButton, QDialog, QGridLayout
+
+from helpers import ParserWindow, config
+
+
+JS_ADD_CSS_TEMPLATE = """(
```

Comment:
TODO: Remove this, didn't end up using any custom CSS. Maybe some tweaking would be good though?